app/[addressOrDomain]/page.tsx

```diff
@@ -362,9 +362,7 @@ export default function Page({ params }: AddressOrDomainProps) {
                 fontFamily: "Sora",
                 minHeight: "32px",
               }}
-              label={`To claim (${
-                claimableQuests ? claimableQuests.length : 0
-              })`}
+              label={claimableQuests && claimableQuests.length > 0 ? `To claim (${claimableQuests.length})` : ""}
```

Comment:
Well, here you're adding the right logic but not at the right place cause you're just deleting the label.

<img width="338" alt="Capture d’écran 2024-08-03 à 15 28 56" src="https://github.com/user-attachments/assets/b63c9230-33c8-4a19-b784-7b1783391681">


Here is what you need to do: 


```tsx
            {claimableQuests.length > 0 ? (
              <Tab
                disableRipple
                sx={{
                  borderRadius: "10px",
                  padding: "0px 12px 0px 12px",
                  textTransform: "none",
                  fontWeight: "600",
                  fontSize: "12px",
                  fontFamily: "Sora",
                  minHeight: "32px",
                }}
                label={`To claim (${claimableQuests})`}
                {...a11yProps(1)}
              />
            ) : null}
            
            ```